packages/app/src/navigation/SidebarNavigationRow.vue

```diff
@@ -48,13 +48,19 @@
         {{ name }}
       </span>
       <span
-        v-if="badge"
+        v-if="badge && !showDot"
         :aria-label="badge.label"
         class="rounded-md font-medium text-white p-4px transition-opacity z-1"
         :class="[badgeVariant, badgeColorStyles[badge.status], {'opacity-0': transitioning}]"
       >
         {{ badge.value }}
       </span>
+      <div
+        v-else-if="badge && showDot"
+        :class="[{'opacity-0': transitioning}, dotClass]"
```

Comment:
Should we just inline the classes? Seems to be the convention, eg

```
class="w-10px ...."
:class="{ opacity-0 ... }"
```

Reply:
I think it's fine to do it this way here since we're calculating the class name in the script. We do the same above for the "badge" piece